ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -10,12 +10,15 @@ def __init__(self):
         self.max_side = 45
 
         # Define minimum and maximum hsv values for red, yellow and green
-        self.lower_red = np.array([0,150,150])
-        self.upper_red = np.array([20,255,255])
+        self.lower_red = np.array([0,120,150])
+        self.upper_red = np.array([25,255,255])
         self.lower_yel = np.array([30,150,150])
         self.upper_yel = np.array([50,255,255])
         self.lower_gre = np.array([60,150,150])
         self.upper_gre = np.array([90,255,255])
+        self.count_thres = 100
+
+        self.counter = 0
```

Comment:
Only used for debugging -> make this somehow clear depending on what you do with the comment below

Reply:
Done